mlflow/server/js/src/model-registry/components/ModelVersionView.js

```diff
@@ -214,14 +214,16 @@ export class ModelVersionViewImpl extends React.Component {
 
   renderCreatorDescription(user_id) {
     return (
-      <Descriptions.Item
-        label={this.props.intl.formatMessage({
-          defaultMessage: 'Creator',
-          description: 'Label name for creator metadata in model version page',
-        })}
-      >
-        {user_id}
-      </Descriptions.Item>
+      user_id && (
```

Comment:
The code usage
```
    const defaultOrder = [
      this.renderRegisteredTimestampDescription(modelVersion.creation_timestamp),
      this.renderCreatorDescription(modelVersion.user_id),
      this.renderStageDropdown(modelVersion),
      this.renderLastModifiedDescription(modelVersion.last_updated_timestamp),
      this.renderSourceRunDescription(),
    ];
 ```
 
 I think the entry in the array here would be undefined, not sure if that is ok to do?
 
 We can also do
 ```
 let defaultOrder = [
      this.renderRegisteredTimestampDescription(modelVersion.creation_timestamp),
      this.renderStageDropdown(modelVersion),
      this.renderLastModifiedDescription(modelVersion.last_updated_timestamp),
      this.renderSourceRunDescription(),
 ];
 
 if(user_id) {
          defaultOrder = [...defaultOrder, this.renderCreatorDescription(modelVersion.user_id),]
 }
 ```
 Not sure what the intended behavior is.

Reply:
I had the same doubt. The unit test was testing this exact behavior. The current code seems to be working fine. Your suggested approach would also work but the order of rendering will deviate from its current behavior. Of course I can make it a joint of 2 or 3 lists to maintain that ordering. What's the preferred JS style in this case?

Reply:
Ahhh that makes sense to maintain ordering. So we can do something like
```
defaultOrder = [... defaultOrder.slice(0, 1), this.renderCreatorDescription(modelVersion.user_id), ...defaultOrder.slice(2)];
```

Reply:
I did this instead:
` return defaultOrder;`
to
` return defaultOrder.filter(x => x !== null);`
Does it work?

Reply:
Yes I think that should work as well